react/modules/inputValueType.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-syntax
```

Comment:
Why can't change this Enum to an object?

Reply:
It breaks a lot of other files, and I didn't found how to migrate it smoothly